### DISCUSSION TITLE
for school pcs - how to install almost any app from microsoft store without access

### DISCUSSION THREAD

**Discussion by Inglan2** [3kh0/ext-remover#1086] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
## Method 1 - winget
open cmd and type `winget search [name of app to install]`
you should get an output of this:
![image](https://github.com/3kh0/ext-remover/assets/117789688/e56aee43-6b34-4b28-8b15-ba501e1a135f)
if that works, use `winget install --id [id of app from results]`

## Method 2 - appx
if that doesnt work use https://github.com/inglan2/appx-downloader on gitpod or codespaces. Instructions are in the repo

Tip:
to check if your school has any monitoring software, press ctrl+shift+esc (if nothing happens they probably blocked task manager). then push more info and then scroll and if you see something like rapid7, ninjaone or anything that doesnt look windowsy its probably monitoring software. you can google it to double check

**Comment by S-PScripts** [comment]
my school banned the settings app on the school computers _**(not our chromebooks)**_ so it's likely that's banned too

**Reply by Inglan2** [reply]
can you open powershell?

**Reply by helloIlovegimkit** [reply]
you forgot another - in -id it should be --id not -id

**Reply by catfoolyou** [reply]
I get this error:

The installer will request to run as administrator, expect a prompt.
An unexpected error occurred while executing the command:
0x8007029c : An assertion failure has occurred.

The "this app is blocked by admin" windows pops up as well

**Reply by helloIlovegimkit** [reply]
yeah there's no patch for the "This app is blocked by admin" if I try to install smth like roblox it's actually blocked since I'm pretty sure they block all the connection services that roblox needs to roblox

**Reply by Inglan2** [reply]
> you forgot another - in -id it should be --id not -id

fixed

> The installer will request to run as administrator, expect a prompt.

This is because the app isnt from the microsoft store. in the source column find one that says msstore

**Reply by Inglan2** [reply]
or try method 2

**Comment by helloIlovegimkit** [comment]
> ## Method 1 - winget
> open cmd and type `winget search [name of app to install]` you should get an output of this: ![image](https://private-user-images.githubusercontent.com/117789688/332613689-e56aee43-6b34-4b28-8b15-ba501e1a135f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY5MTUzMDYsIm5iZiI6MTcxNjkxNTAwNiwicGF0aCI6Ii8xMTc3ODk2ODgvMzMyNjEzNjg5LWU1NmFlZTQzLTZiMzQtNGIyOC04YjE1LWJhNTAxZTFhMTM1Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyOFQxNjUwMDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iODgwMzJjN2MyNmMzYmVjOWZhNzBhOWQ3Zjg0OGFiZWE0MWUxMjZlZjg4YmMyMTA5NzQyZDVjNzE1YzVhMTlmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.IZflTSuffVB71nUF3JActv68QEuQ82R1WGFIShjx1BE) if that works, use `winget install --id [id of app from results]`
> 
> ## Method 2 - appx
> if that doesnt work use https://github.com/inglan2/appx-downloader on gitpod or codespaces. Instructions are in the repo
> 
> Tip: to check if your school has any monitoring software, press ctrl+shift+esc (if nothing happens they probably blocked task manager). then push more info and then scroll and if you see something like rapid7, ninjaone or anything that doesnt look windowsy its probably monitoring software. you can google it to double check



You could though your school/district has to be stupid to let you (mine is) you can download python and use git works for me because my school's security is idiotic

**Reply by helloIlovegimkit** [reply]
For python you have to add it to PATH and if your school is stupid enough (mine is) to let you just access the school's own account then yea you could maybe add it to PATH, though at that point just use the winget method

**Reply by Inglan2** [reply]
our school has python installed but we cant use git. we have all the monitoring software and the computers are slow asf. maybe the chromebooks are better...
